frontera/contrib/backends/remote/codecs/msgpack.py

```diff
@@ -30,7 +30,8 @@ def serialize(obj):
         elif hasattr(obj, '__dict__'):
             return serialize(obj.__dict__)
         else:
-            logger.warning('unable to serialize object: {}'.format(obj))
+            if obj:
```

Comment:
Thanks for your reply!  But I still have some question?

> However your fix doesn't seem right to me. What happens when obj=True or say there is a key in a dict which has a value that is True? It will still output the warning message in such a case.

When _obj=True_ , isn't the program going to this condition  ? 
```
if isinstance(obj, (bool, six.integer_types, float, six.binary_type, six.text_type)):
    return obj
```

Another question, will you not fix this bug?